--- src/main/resources/schemas/v1.2-rc1/cybox/objects/Archive_File_Object.xsd	2014-12-10 15:08:05.000000000 -0500
+++ src/main/resources/schemas/v1.2-rc1/cybox/objects/Patched_Archive_File_Object.xsd	2014-12-11 12:21:38.000000000 -0500
@@ -39,16 +39,6 @@
                             <xs:documentation>The File_Count field specifies the number of files contained within the archive.</xs:documentation>
                         </xs:annotation>
                     </xs:element>
-                    <xs:element minOccurs="0" name="Encryption_Algorithm" type="cyboxCommon:CipherType">
-                        <xs:annotation>
-                            <xs:documentation>The Encryption_Algorithm field specifies the algorithm used to encrypt an archive file.  Note: For individual files within an archive that are themselves encrypted, the specifics of that encryption are found in the FileObject definition for that file.</xs:documentation>
-                        </xs:annotation>
-                    </xs:element>
-                    <xs:element name="Decryption_Key" type="cyboxCommon:StringObjectPropertyType" minOccurs="0">
-                        <xs:annotation>
-                            <xs:documentation>The Decryption_Key field specifies the key used to decrypt the file.</xs:documentation>
-                        </xs:annotation>
-                    </xs:element>
                     <xs:element minOccurs="0" name="Comment" type="cyboxCommon:StringObjectPropertyType">
                         <xs:annotation>
                             <xs:documentation>The Comment field specifies the comment information associated with archive file.</xs:documentation>
